gdx/src/com/badlogic/gdx/scenes/scene2d/ui/List.java

```diff
@@ -171,7 +174,8 @@ public void draw (Batch batch, float parentAlpha) {
 					selectedDrawable.draw(batch, x, y + itemY - itemHeight, width, itemHeight);
 					font.setColor(fontColorSelected.r, fontColorSelected.g, fontColorSelected.b, fontColorSelected.a * parentAlpha);
 				}
-				drawItem(batch, font, i, item, x + textOffsetX, y + itemY - textOffsetY);
+				layout.setText(font, toString(item), 0, toString(item).length(), font.getColor(), width, alignment, false, "...");

```

Comment:
You no longer call `drawItem`, which subclasses may rely on. Better to add `width` and `alignment` parameters to `drawItem`. Also, a GlyphLayout should not be needed, BitmapFont has methods to draw with alignment (which internally uses a temp GlyphLayout).